modules/icgc-argo-workflows/checkinput/main.nf

```diff
@@ -0,0 +1,56 @@
+process SAMPLESHEET_CHECK {
+    tag "$samplesheet"
+    label 'process_single'
+
+    conda "conda-forge::python=3.8.3"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/python:3.8.3' :
+        'quay.io/biocontainers/python:3.8.3' }"
+
+    input:
+    path samplesheet
+    val workflow_name
+
+    output:
+    path '*.csv'       , emit: csv
```

Comment:
should we make it specific to `samplesheet.valid.csv`?

Reply:
Addressed:
https://github.com/icgc-argo-workflows/argo-modules/blob/e7871dd8ba5a717740a230c2487f3e3de3492f04/modules/icgc-argo-workflows/checkinput/main.nf#L15